metrics/metrics.go

```diff
@@ -18,12 +18,16 @@ package metrics
 
 import (
 	"github.com/containerd/containerd/version"
-	goMetrics "github.com/docker/go-metrics"
+	metrics "github.com/docker/go-metrics"
+)
+
+var (
+	buildInfoLabeledCounter metrics.LabeledCounter
```

Comment:
What's this change?

Reply:
At first, GCCounter was defined at here, so i extracted the buildinfo label's declaration to here.